src/sentry/utils/committers.py

```diff
@@ -293,7 +293,7 @@ def get_event_file_committers(
 
 
 def get_serialized_event_file_committers(
-    project: Project, event: Event, frame_limit: int = 25
+    project: Project, event: Event | GroupEvent, frame_limit: int = 25
```

Comment:
Since both of these are subclasses of `BaseEvent`, could we loosen the type to the parent by chance?